src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -399,7 +399,14 @@ public void close(Collection<TopicPartition> partitions) {
   }
 
   @Override
-  public void setIsStoppedToTrue() {}
+  public void stop() {
+    try {
+      StreamingClientProvider.getStreamingClientProviderInstance()
+          .closeClient(this.connectorConfig, this.streamingIngestClient);
```

Comment:
the client can be shared across multiple sink connectors, so calling stop here may break or cause a new client to open on other connectors. 
Please try running this code with multiple connectors pointing to one client to understand the behavior. And/or write a test with multiple sinks pointing to one client as well